disperser/cmd/apiserver/main.go

```diff
@@ -101,6 +103,37 @@ func RunDisperserServer(ctx *cli.Context) error {
 
 	reg := prometheus.NewRegistry()
 
+	var meterer *mt.Meterer
+	if config.EnablePaymentMeterer {
+		mtConfig := mt.Config{
+			ChainReadTimeout: 3 * time.Second,
+		}
+
+		paymentChainState, err := mt.NewOnchainPaymentState(context.Background(), transactor)
+		if err != nil {
+			return fmt.Errorf("failed to create onchain payment state: %w", err)
+		}
+
+		offchainStore, err := mt.NewOffchainStore(
+			config.AwsClientConfig,
+			"reservations",
```

Comment:
Are these switched to constants in a future PR by chance?

Reply:
nope, I updated to use config flag (optional with defaults)

Reply:
should these be read from config?